cg/meta/workflow/mip.py

```diff
@@ -13,7 +13,7 @@
 from cg.meta.deliver import DeliverAPI
 from cg.store import models, Store
 from cg.exc import CgDataError, LimsDataError
-from cg.constants import COMBOS, COLLABORATORS, MASTER_LIST, DEFAULT_CAPTURE_KIT
+from cg.constants import COMBOS, COLLABORATORS, MASTER_LIST, DEFAULT_CAPTURE_KIT, FAMILY_ACTIONS
```

Comment:
Nice! Could we use CASE_ACTIONS? We are trying to move away from family as much as possible

Reply:
This is an old constant, it would affect a lot of code! Perhaps a story for another PR?